### PR TITLE
DOCS-622: Improve Micro-rdk and rdk

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/internals/micro-RDK.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "micro-RDK"
+linkTitle: "micro-RDK"
+weight: 2
+type: "docs"
+description: "The micro-RDK is a lightweight version of the Robot Development Kit which can run on resource-limited embedded systems."
+tags: ["server", "rdk"]
+---
+
+{{% readfile "/static/include/micro-rdk.md" %}}
```
Comment (npentrel) [inline docs/internals/micro-RDK.md:10]:
```suggestion
{{% readfile "/static/include/micro-rdk.md" %}}

```

File: static/include/micro-rdk.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+{{% /alert %}}
+
+The micro-RDK is a lightweight version of the {{% glossary_tooltip term_id="rdk" text="Robot Development Kit (RDK)"%}} which can run on resource-limited embedded systems.
+
+The only microcontroller the micro-RDK currently supports is the [ESP32](https://www.espressif.com/en/products/socs/esp32).
+
+To use Viam on a microcontroller, you need to:
+
+- run the micro-RDK on your microcontroller
+- run the full-featured `viam-server` on another machine
```
Comment (JessamyT) [inline static/include/micro-rdk.md:14]:
```suggestion
- Run the micro-RDK on your microcontroller
- Run the full-featured `viam-server` on another machine
```
Maybe: add and "and" in there to make it extra explicit?

**Review by JessamyT — COMMENTED** [review]
Seems good. Commented on some tiny nits. But main question is why micro rdk is becoming an includes instead of a page like RDK

File: static/include/micro-rdk.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+The micro-RDK is in alpha mode and many features supported by the RDK are still being added to the micro-RDK.
+Stability is not guaranteed.
+Breaking changes are likely to occur, and occur often.
+{{% /alert %}}
+
+The micro-RDK is a lightweight version of the {{% glossary_tooltip term_id="rdk" text="Robot Development Kit (RDK)"%}} which can run on resource-limited embedded systems.
+
+The only microcontroller the micro-RDK currently supports is the [ESP32](https://www.espressif.com/en/products/socs/esp32).
+
+To use Viam on a microcontroller, you need to:
```
Comment (npentrel) [inline static/include/micro-rdk.md:11]:
```suggestion
To use Viam on a microcontroller, you need to both:
```

**Review by JessamyT — APPROVED** [review]
LGTM % don't know why it's just "Next" instead of "Next steps" like on many other pages. Or "See also." "Next" just seems awk to me

**Pull request merged**